modules/javafx.graphics/src/main/java/com/sun/marlin/MarlinProperties.java

```diff
@@ -268,30 +266,20 @@ public static float getQuadDecD2() {
     }
 
     // system property utilities
-    @SuppressWarnings("removal")
```

Comment:
really, this should be in Utils.

Reply:
Unrelated (but yes, we have several flavors of similar utilities that could be unified in some future cleanup issue).

Reply:
I wrote all needed helper when marlin was a side-project (plugin), but I totally agree dealing easily with such config files should have been provided by jdk.internals.config classes

Reply:
We can't use any JDK internals in JavaFX, so if we ever do anything about this, we would create methods in a utility class in a non-exported package in `javafx.base`.